integration/simple_npm_app_test.go

```diff
@@ -0,0 +1,69 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testSimpleNPMApp(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		pack occam.Pack
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack()
+	})
+
+	context("when building a simple npm app", func() {
+		var (
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(os.RemoveAll(source)).To(Succeed())
```

Comment:
I'm not sure I understand what you mean - the test doesn't make use of the image so the normal `docker.x.Remove.Execute(y)` didn't seem necessary. Looking through the currently implemented occam stuff, there's only random string generation (`occam.RandomName()`) and a temporary directory (source) that is deleted correctly, I believe - am I missing something?

Reply:
addressed [here](https://github.com/paketo-buildpacks/node-run-script/pull/1/commits/ea27a66f65cc45a02785985e7a7769402645600b#diff-d5321cf60f24e61b6c00f3031808bc0e1c7ede6890feb0f4c1e0197d6fc274f0R46)?

Reply:
@AccraZed Regardless of whether the test runs a container out of the image, the image is created on the docker daemon where the test is run. The test code should store a reference to it in a variable and later clean it up. Otherwise your test will start leaving substantial amount of bloat on the user's machine every time the test is run. You can run `docker image ls -a | sort` before and after the test and make sure that you don't litter your test images.